pkg/controller/dmcluster/dm_cluster_controller.go

```diff
@@ -245,6 +253,12 @@ func (dcc *Controller) sync(key string) error {
 	if err != nil {
 		return err
 	}
+	clusterVersionLT2, err := clusterVersionLessThan2(dc.MasterVersion())
+	if err != nil {
+		klog.V(4).Infof("cluster version: %s is not semantic versioning compatible", dc.MasterVersion())
+	} else if clusterVersionLT2 {
+		return fmt.Errorf("dm-operator only supports to deploy dm-2.0")
```

Comment:
Return error here will cause it to resync soon, I think we can log the error here and just return nil.

Reply:
OK